packages/archetype/src/__tests__/__snapshots__/Archetype.spec.js.snap

```diff
@@ -125,19 +125,145 @@ exports[`Archetype Default 1`] = `
       }
     }
   >
-    <Example>
-      <div>
-        <Archetype>
-          <glamorous(div)>
-            <div
-              className="css-nil"
-            >
-              Archetype
-            </div>
-          </glamorous(div)>
-        </Archetype>
+    <LiveProvider
```

Comment:
LivePreview is needed to render the string of source, but there are no styles attached to the snapshot tests, see `testDemoExamples.js`. We're just sending the source string into an unstyled LiveProvider and LivePreview so that they get rendered. 

The styles for the demo are in ComponentDocExample and are separate. You can modify those styles without affecting the snapshot tests. 

What will affect the snapshot tests is if you modify the styles on the demo examples, for instance on the layout components wrapping several buttons in a single example.